app/src/main/java/com/example/rajadav/adidas/DetailActivity.java

```diff
@@ -0,0 +1,24 @@
+package com.example.rajadav.adidas;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.TextView;
+
+public class DetailActivity extends AppCompatActivity {
```

Comment:
Add an small description telling future devs (or future you) what this activity is doing